src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -126,6 +127,45 @@ class C
             var exitCode = cmd.Run(outWriter);
             Assert.Equal(0, exitCode);
             Assert.Equal("", outWriter.ToString());
+
+            var compilerTreeOptions = Assert.IsType<CompilerPerTreeOptionsProvider>(cmd.TreeOptionsProvider);
+            Assert.True(compilerTreeOptions.IsEmpty);
```

Comment:
Why the change? Won't Assert.Empty give better diagnostics if it were to fail?

Reply:
This type doesn't implement IEnumerable or any other interface.